react/components/GalleryItem.js

```diff
@@ -25,12 +26,22 @@ export default class GalleryItem extends Component {
     const normalizedProduct = { ...product }
     const [sku] = normalizedProduct.items || []
 
+    const sum = array => array.reduce((x, y) => x + y)
+
+    const transform = array => array.map(item => {
```

Comment:
Use Ramda's map and path. It would be like

`const transform = map(path(['sellers', '0', 'commertialOffer', 'AvailableQuantity']))`